bitnami/fluentd/templates/aggregator-statefulset.yaml

```diff
@@ -29,6 +29,7 @@ spec:
         {{- end }}
     spec:
       {{- include "fluentd.imagePullSecrets" . | nindent 6 }}
+      serviceAccountName: {{ template "fluentd.aggregator.serviceAccountName" . }}
```

Comment:
Shouldn't this property be added conditionally, depending on whether service accounts are enabled or not for the aggregator?

Reply:
Hi @marcosbc - this is following the same style as  the forwarder does - if the serviceAccounts are not created by the chart or the `serviceAccount.name` is not overridden, the value [off the template here](https://github.com/bitnami/charts/pull/3293/files#diff-f184e19aae77c89089cfe28594781b45R111) will be `default` which is the same as what pods will get out of the box if you don't set the value in the pod spec.

Reply:
In any case, I have reverted to the original name of `serviceaccount.yaml` as this seems the most consistent with the most number of charts.

Reply:
I think this is ok thanks!